### ISSUE TITLE
The default size of the Properties dialog is too small

### ISSUE THREAD

**Issue by v-xuanzh** [Microsoft/AzureStorageExplorer#1245]
**Storage Explorer Version:** 1.7.0_20190319.1
**Platform/OS:** Linux Ubuntu 18.04
**Architecture:** x64
**Regression From:** Not a regression

**Steps to reproduce**:
1. Expand any one account -> 'Blob Containers'.
2. Open a blob contain which contains one file at least.
3. Add one property for the blob -> Check the layout of the Properties dialog.

**Expect Experience:**
1. The 'Properties' dialog shows well.
2. No scrollbar display.

**Actual Experience:**
1. The default size is too small.
![image](https://user-images.githubusercontent.com/34729022/54670354-b7cee580-4ab0-11e9-8509-3ccb0b7a20ec.png)
2. Need to drag the scrollbar to see all the contents.

**More info:**
1. This issue also reproduces for File/ADLS Gen2 files dialog.
2. This issue doesn't reproduce on Ubuntu 16.04

**Comment by MRayermannMSFT** [comment]
Hey @v-xuanzh , I'm unable to repro this:
![image](https://user-images.githubusercontent.com/21112697/55181379-cbc9b580-5148-11e9-9fd1-597d9480e213.png)
Can you confirm on latest build? And if you still see it, please let me know your display setting:
![image](https://user-images.githubusercontent.com/21112697/55181468-f74ca000-5148-11e9-8151-af524f56588c.png)

**Comment by v-xuanzh** [comment]
Hi @MRayermannMSFT , Looks like you have not added properties. 
This issue can be reproduced after added some properties. So reopened it.

**Comment by v-xuanzh** [comment]
Hi @MRayermannMSFT , I have verified this issue with the latest master build. And for now, The scrollbar shows on the metadata field. Is it has been fixed?
![image](https://user-images.githubusercontent.com/34729022/56177076-3fc1e580-5fb2-11e9-8623-c951b670922d.png)

**Comment by MRayermannMSFT** [comment]
We're consider this as fixed for now since the scroll bar you are seeing only repro's on Ubuntu 18. If we get customer feedback about it then we will revisit.
